test/literature/pagination_test.exs

```diff
@@ -0,0 +1,174 @@
+defmodule Literature.PaginationTest do
+  use Literature.DataCase
+
+  import Literature.Test.Fixtures
+
+  alias Literature.Pagination
+  alias Literature.Post
+
+  @pagination_params %{page: 2, page_size: 4}
+
+  setup :setup_posts
+
```

Comment:
We need an additional setup with an `on_exit/2` callback to put the original config back so it will not affect other test modules:
```suggestion
  setup :setup_posts
  
  setup do
    original_config = Application.get_all_env(:flop)
    on_exit(fn -> Application.put_all_env(flop: original_config) end)
  end
```
Lately, we just pushed this change to the web app https://github.com/martide/elixir/pull/8750/files